bin/ember

```diff
@@ -33,3 +38,16 @@ resolve('ember-cli', {
     exit(exitCode);
   });
 });
+
+function notifyUpdate(update) {
+  if (!process.stdout.isTTY || !update || require('is-npm')) {
+    return;
+  }
+
+  var chalk = require('chalk');
+
+  var message = '\nA new ' + chalk.yellow('ember-cli') + ' release is available (' + update.current + ' → ' + chalk.yellow(update.latest) + ')\n' +
+    'Visit ' + chalk.yellow.underline('https://github.com/ember-cli/ember-cli/releases/tag/v' + update.latest) + ' for update instructions\n';
+
+  console.error(message);
```

Comment:
I'm not 100% sure that our test setup at work doesn't shell out to `ember test` since we mate a whole bunch of commands to run our tests and rely on `&&` to join them. `npm run test` could also blow up with an error code exit.